generator/feature_sorter.cpp

```diff
@@ -371,6 +374,10 @@ bool GenerateFinalFeatures(feature::GenerateInfo const & info, string const & na
         collector(GetFeatureBuilder2(f));
       }
 
+      // Update bounds with limit rect corresponding region borders.
+      // Bounds before update can be too big because of big invisible features like country
```

Comment:
like a country border relation
like a relation that contains an entire country's border

Reply:
done